pkg/antctl/raw/proxy/command.go

```diff
@@ -61,6 +61,7 @@ type proxyOptions struct {
 }
 
 var options *proxyOptions
+var fsTest = afero.NewOsFs()
```

Comment:
`fsTest` is not a great name (even if tests are the reason for introducing this variable, as the name would imply it is only used for testing). I see that we use `defaultFS` several times in the code base, for the same purpose.

Reply:
ok, will change it.